cdap-docs/developer-manual/source/cloud-runtimes/provisioners/gcp-dataproc.rst

```diff
@@ -8,5 +8,95 @@
 Google Dataproc
 ===============
 
-Describe what Dataproc is, then describe each config setting
+Dataproc is a Google Cloud Platform (GCP) service that manages Hadoop clusters in the cloud.
+It can be used to create large clusters in around one minute.
+The Google Dataproc provisioner simply calls the Dataproc APIs in order to create and delete
+clusters in your GCP account. 
+The provisioner exposes several configuration settings that control what type of cluster is created.
 
+Account Information
+-------------------
+
+Project ID
+^^^^^^^^^^
+A GCP project ID must be provided. This will be the project that the Dataproc cluster is created in.
+The project must have the Dataproc APIs enabled.
+
+Service Account Key
+^^^^^^^^^^^^^^^^^^^
+A service account key is a secret that allows automated processes make requests against the GCP APIs.
```

Comment:
The service account key provided to the provisioner must have rights to access the Cloud Dataproc APIs and the Google Compute Engine APIs. Since your account key is sensitive, we recommended that you provide your account key through the CDAP :ref:`Secure Storage <http-restful-api-secure-storage>` by adding a secure key with the RESTful API and clicking the shield icon in the UI to select a secure key.